qa/rpc-tests/segwit_recovery.py

```diff
@@ -9,6 +9,8 @@
 """
 
 import time
+from test_framework.util import *
```

Comment:
why importing everything from `util`? I'd rather prefer adding what's actually needed to the `import` as stated at L41.

Reply:
fixed